mw-webapp/src/pages/reportsTable/ReportsTable.tsx

```diff
@@ -0,0 +1,46 @@
+import {useMemo} from "react";
+import {getCoreRowModel, HeaderGroup, RowModel, useReactTable} from "@tanstack/react-table";
+import {columns} from "src/component/table/columns";
+import {Table} from "src/component/table/Table";
+import {useGetDataTable} from "src/dataAccessLogic/useGetDataTable";
+import {DayReport} from "src/model/businessModel/DayReport";
+
+
+/**
+ * Props table of reports
```

Comment:
if it is props call interface TableReportsProps

Reply:
done